src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -166,7 +189,6 @@
                                     x:Uid="RemoveButton"
                                     Background="Transparent"
                                     FontFamily="Segoe MDL2 Assets"
-                                    Height="34"
```

Comment:
I'd go for MinHeight instead of removing it entirely to make sure it renders properly when setting display scaling to e.g. 100%/150%